tests/sentry/issues/endpoints/test_organization_group_search_views.py

```diff
@@ -103,3 +106,198 @@ def test_get_default_views(self):
         assert view["query"] == "is:unresolved issue.priority:[high, medium]"
         assert view["querySort"] == "date"
         assert view["position"] == 0
+
+
+class OrganizationGroupSearchViewsPutTest(APITestCase):
+    endpoint = "sentry-api-0-organization-group-search-views"
+    method = "put"
+
+    def create_base_data(self):
+        self.custom_view_one = GroupSearchView.objects.create(
+            name="Custom View One",
+            organization=self.organization,
+            user_id=self.user.id,
+            query="is:unresolved",
+            query_sort="date",
+            position=0,
+        )
+
+        self.custom_view_two = GroupSearchView.objects.create(
+            name="Custom View Two",
+            organization=self.organization,
+            user_id=self.user.id,
+            query="is:resolved",
+            query_sort="new",
+            position=1,
+        )
+
+        self.custom_view_three = GroupSearchView.objects.create(
+            name="Custom View Three",
+            organization=self.organization,
+            user_id=self.user.id,
+            query="is:ignored",
+            query_sort="freq",
+            position=2,
+        )
+
+        return {
+            "views": [
+                self.custom_view_one,
+                self.custom_view_two,
+                self.custom_view_three,
+            ]
+        }
+
+    def setUp(self):
+        self.login_as(user=self.user)
+        self.base_data = self.create_base_data()
+
+        self.url = reverse(
+            "sentry-api-0-organization-group-search-views",
+            kwargs={"organization_id_or_slug": self.organization.slug},
+        )
+
+    @with_feature({"organizations:issue-stream-custom-views": True})
```

Comment:
might be going crazy but i feel like most of the time i see `with_feature` i just see it as `@with_feature("organizations:issue-stream-custom-views")` . is there a reason why you have it this way?

Reply:
Nope I have no idea why I made it that way either - probably just copy pasted it from somewhere else. Looks like it still works as expected though